index.js

```diff
@@ -11,11 +12,46 @@ server.get('/payments', (req, res) => {
     res.status(200).send(payments);
 });
 
+server.get('/payments/:id', (req, res) => {
+    const payment = payments.find( payment => payment.id === req.params.id);
+    if(payment){
+        res.status(200).send(payment);
+    }
+    else{
+        res.status(404).send({message: 'Wrong id'});
+    }
+});
+
+server.post('/payments', (req, res) => {
+    const payment = {...req.body};
+    Object.assign(payment, {
+        id: uuidv4(),
+        status: 'Pending'
+    });
+    payments.push(payment);
+    res.status(200).send(payment);
+});
 
+server.put('/payments/:id', (req, res) => {
+    const payment = payments.find( payment => payment.id === req.params.id);
+    if(req.body.description){
+        payment.description = req.body.description;
+        res.status(200).send(payment);
+    }
+    else{
+        res.status(404).send('Wrong description')
```

Comment:
hi
please refer to the doc linked below to choose better http status code for the response in the event when we are getting wrong request. 
http status code of 404 is a bit different thing ;)
 https://developer.mozilla.org/en-US/docs/Web/HTTP/Status